Packages/ChineseNumbers/Sources/ChineseNumbers/SuzhouNumbers.swift

```diff
@@ -0,0 +1,71 @@
+import Foundation
+
+public class SuzhouNumbers: NSObject {
+
+    static private let verticalDigits: [Character: String] = [
+        "0": "〇", "1": "〡", "2": "〢", "3": "〣", "4": "〤",
+        "5": "〥", "6": "〦", "7": "〧", "8": "〨", "9": "〩"
+    ]
+    static private let horizontalDigits: [Character: String] = [
+        "1": "一", "2": "二", "3": "三"
+    ]
+    static private let placeNames = [
+        "", "十", "百", "千",
+        "万", "十万", "百万", "千万",
+        "億", "十億", "百億", "千億",
```

Comment:
AFAIK, people use 万 when doing 苏州码. It is part of the ancient number system, and it is not about Simplified or Traditional Chinese. Actually I do not think people may use 苏州码 to present numbers larger than 億 but I just added it since I allowed users to input such a big number.